src/api/productAPI.ts

```diff
@@ -18,7 +16,7 @@ const baseURL = process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL;
 
 export async function getAllProductList({ sort, page, size }: ProductParams): Promise<ProductListResponse> {
   try {
-    const response = await fetch(`${baseURL}/api/v1/product/get/all-list?&sort=${sort}&page=${page}&size=${size}`);
+    const response = await fetch(`${baseURL}/api/v1/product/all?&sort=${sort}&page=${page}&size=${size}`);
```

Comment:
```suggestion
    const response = await fetch(`${BASE_URL}/api/v1/product/all?&sort=${sort}&page=${page}&size=${size}`);
```
도원님 작업하신내용은 아닌거같은데 baseURL 변수명은 저희 컨벤션이랑 맞않아보이는군요